staging/src/k8s.io/apiserver/pkg/server/routes/openstack.go

```diff
@@ -67,7 +71,7 @@ func isBatchCreationRequest(req *http.Request) (bool, error) {
 	err = json.Unmarshal(body, &obj)
 
 	if err != nil {
-		klog.Errorf("error unmarshal request. error %v", err)
+		klog.Errorf("error unmarshal request body :%s. error %v", string(body), err)
```

Comment:
This can easily overflow the log.

Reply:
Good catch. that could be true. this is added for debugging issues i will separate the log body to a level controlled info log.

Reply:
fixed.